compute_endpoint/globus_compute_endpoint/endpoint/endpoint.py

```diff
@@ -133,7 +132,9 @@ def configure_endpoint(
         print(
             f"\n    Configuration file: {config_path}\n"
             "\nUse the `start` subcommand to run it:\n"
-            f"\n    $ funcx-endpoint start {ep_name}"
+            f"\n    $ globus-compute-endpoint start {ep_name}"
+            # If we want to show 'funcx-endpoint start' when using the wrapper
+            # f"\n    $ {sys.argv[0]} start {ep_name}"
```

Comment:
It's a good thought, but I don't think we do.  It _should_ work, but we're rebranding to Globus Compute, so I'm thinking that the output is always `$ globus-compute-endpoint start ...` regardless.

Reply:
Removed